src/bin/package_for_release/bundle_linux.rs

```diff
@@ -31,11 +35,14 @@ pub fn bundle_linux_app() -> Result<()> {
         release_dir.join("main_process"),
         tmp_dir.join("video_compositor_main"),
```

Comment:
```suggestion
        tmp_dir.join("video_compositor"),
```

and do not copy `compositor_runtime_wrapper` and `lib` bellow